### PR TITLE
fix(viewer): set correct rgba value (DSP-1588)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DSP-1588

### CODE REVIEW

File: projects/dsp-ui/src/lib/viewer/values/color-value/color-value.component.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
 }
 
 .color-preview {
-  border: 1px solid rgba(33,33,33.5);
+  border: 1px solid rgba(33,33,33,.5);
```
Comment (flavens) [inline projects/dsp-ui/src/lib/viewer/values/color-value/color-value.component.scss:19]: .5 or 0.5??

**Review by mdelez — APPROVED** [review]
NICE!

File: projects/dsp-ui/src/lib/viewer/values/color-value/color-value.component.scss (hunk 1 above)
Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/values/color-value/color-value.component.scss:19]: yes, in scss we don't need a leading 0

Comment (kilchenmann) [reply projects/dsp-ui/src/lib/viewer/values/color-value/color-value.component.scss:19]: https://github.com/sds/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero

**Pull request merged**